app/photo.py

```diff
@@ -40,6 +47,20 @@ class SizeTooLargeError(PhotoError):
                 'Please upload a smaller one.')
 
 
+def create_photo_from_url(handler, photo_url):
```

Comment:
`create_photo` and `create_photo_from_url` have `handler` in different positions. For consistency consider swapping to handler always first.

Reply:
Done.